src/bust.py

```diff
@@ -325,6 +326,40 @@ def get_google_form_url(self, image_url: Optional[str] = None) -> Optional[str]:
         )
         return form_url
 
+    async def send_stats(self, interaction: Interaction):
+        songs_len = int(self.total_song_len)
+        num_songs = len(self.current_channel_content)
+        bust_len = songs_len + config.seconds_between_songs * num_songs
+
+        # Compute map submitter --> total length of all submissions
+        submitter_to_len = defaultdict(lambda: 0.0)
+
+        for submit_message, attachment, local_filepath in self.current_channel_content:
+            song_len = song_utils.get_song_length(local_filepath)
+            submitter = submit_message.author
+            submitter_to_len[submitter] += song_len
+
+        # User with longest total submission length
+        longest_submitter = max(submitter_to_len, key=submitter_to_len.get)
+        longest_submitter_time = int(submitter_to_len[longest_submitter])
+
+        embed_text = "\n".join(
+            [
+                f"*Number of tracks:* {num_songs}",
+                f"*Total track length:* {song_utils.format_time(songs_len)}",
+                f"*Total bust length:* {song_utils.format_time(bust_len)}",
+                f"*Unique submitters:* {len(submitter_to_len)}",
+                f"*Longest submitter:* {longest_submitter.mention} : "
```

Comment:
```suggestion
                f"*Longest submitter:* {longest_submitter.mention} - "
```

nit: since we already have a colon on this line.